### ISSUE TITLE
GGMAX particle emitter enhancement needed =

### ISSUE THREAD

**Issue by imothep85** [TheGameCreators/GameGuruRepo#5529]
When i'm adding particles to my level, we have some bugs

1 the PARTICLE generator doesn't GENERATE the particle preview when saving.... 
still not fixed after months of reporting that problem.
2 The particle emitter is TOO BIG & TOO VISIBLE i have to place small fire flames inside my torches, but the emitter is SO big and so visible opaque that it's IMPOSSIBLE to see the base of my flame SO i have to launch the level to test what i see, THEN go back to the game editor, adjust the emitter, run the game again etc...
is it possible to make the red emitter more TRANSPARENT, also maybe its possible to change the shape of the emitter ?
i prefer to have a triangle, the TOP of the triangle/ cone, represent the beginning of the particle, where it start.
![particle emitter](https://github.com/TheGameCreators/GameGuruRepo/assets/8946215/4d4d70c4-af21-438f-870c-0a310d8d9a74)

**Comment by MonkeyFrogStudio** [comment]
#2 is already a suggestion here on GitHub:
https://github.com/TheGameCreators/GameGuruRepo/issues/3741

You can already hide the lights in the editor (and still see their effect). On the left side, below the Level Objects area, is a bar entitled, "View Options". Twirl that down to expose the options there. The first set of tick boxes are called 'Game Elements'. The player start marker, zone markers, lights, etc., are all game elements. Untick the tick box under Editor and all those elements will be hidden until you tick the box again.

**Comment by imothep85** [comment]
yes i know that we can hide the lights , but it HIDE also the emitters, i need to hide the lights to only place the emitters and see my particles
